src/js/components/Nav/Nav.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { Anchor } from '../Anchor';
+import { Box } from '../Box';
+
+// TODO resolve whether children should be allowed with items
+const Nav = ({ children, items = [], ...rest }) => (
+  <Box as="nav" flex={false} gap="medium" {...rest}>
+    {items.map(item => (
+      <Anchor href={item.href} label={item.label} key={item.label} />
```

Comment:
How would the caller integrate this with a router?

Reply:
Not sure why should she/he integrate with a router. 
I think the Nav should be separated from the routing logic, just like Anchor/Button doesn't know which router you are using, and just direct to the `href` provided. 
I might be missing something though, so feel free to elaborate.

Reply:
The routers have to tie in to the `onClick` property. So, you typically want to do something like `<RouterAwareAnchor path="/internal/path" label="Name" />` with the router taking care of `onClick` and `href`.

Reply:
I made the component less opinionated by removing `items`, I think it should solve the routing integration issue.